tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -185,14 +193,13 @@ def _resnet_model_fn(features, labels, mode, params):
           'accuracy':
               tf.metrics.accuracy(stacked_labels, predictions['classes'])
       }
-      loss = tf.reduce_mean(tower_losses, name='loss')
 
     return tf.estimator.EstimatorSpec(
         mode=mode,
         predictions=predictions,
         loss=loss,
         train_op=train_op,
-        training_chief_hooks=chief_hooks,
```

Comment:
Just curious, does this mean that previously non-chief-workers did not have sync replicas optimizer initialized with a hook?  Just based on the name.

Reply:
Yep that's correct